lib/opensusebasetest.pm

```diff
@@ -241,8 +241,13 @@ sub investigate_yast2_failure {
     );
     # Hash with known errors which we don't want to track in each postfail hook
     my %y2log_known_errors = (
-        "<3>.*no[t]? mount"   => 'bsc#1092088',                   # Detect not mounted partition
-        "<3>.*lib/cheetah.rb" => 'bsc#1153749',
+        "<3>.*QueryWidget failed.*RichText.*VScrollValue" => 'bsc#1167248',
+        "<3>.*Solverrun finished with an ERROR"           => 'bsc#1170322',
+        "<3>.*3 packages failed.*badlist"                 => 'bsc#1170322',
+        "<3>.*Unknown option.*MultiSelectionBox widget"   => 'bsc#1170431',
+        "<3>.*XML.*Argument.*to Read.*is nil"             => 'bsc#1170432',
+        "<3>.*no[t]? mount"                               => 'bsc#1092088',    # Detect not mounted partition
+        "<3>.*lib/cheetah.rb"                             => 'bsc#1153749',
```

Comment:
is this really different from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/7e578fe4d5ad53cf12c6a83f2545c2a697c2190c/lib/opensusebasetest.pm#L266?

Reply:
idn, it was there, github sometimes displays diff like you are deleting something but not. This mechanism with regex is a little tricky, I also noticed that would be easy to add something twice by mistake, so it would require some clean up in the future. I'm not planning to investigate existing known errors in the scope of this PR.

Reply:
It's good point. After reading discussion in the bug, it's not clear how yast team wants to proceed with it. If bug gets resolved, we can put this more generic regexp to ignore all cheetah error messages with trello card reference.